packages/parse5/docs/index.md

```diff
@@ -1,111 +0,0 @@
-# parse5
```

Comment:
Should all this information be moved to the readme?

Reply:
The READMEs should definitely be extended a bit. Generated docs like this could also just live in their own space, eg. a website.